### PR TITLE
Add `datana-gmbh/logz-io-handler`

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/datana-gmbh/logz-io-handler

### CODE REVIEW

**Review by OskarStark — COMMENTED** [review]
Note sure about the error, as it worked locally 🤷‍♂️

File: datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+services:
+    Inpsyde\LogzIoMonolog\Handler\LogzIoHandler:
+        bind:
```
[the remark this answers is not in the document]
Comment (OskarStark) [reply datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml:3]: In this case I need to remove the typehint too

File: datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+services:
+    Inpsyde\LogzIoMonolog\Handler\LogzIoHandler:
+        arguments:
+            string $token: '%env(LOGZ_IO_TOKEN)%'
+            string $host: !php/const Inpsyde\LogzIoMonolog\Handler\LogzIoHandler::HOST_EU
```
Comment (OskarStark) [inline datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml:5]:
```suggestion
           $token: '%env(LOGZ_IO_TOKEN)%'
           $host: !php/const Inpsyde\LogzIoMonolog\Handler\LogzIoHandler::HOST_EU
```

File: datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+services:
+    Inpsyde\LogzIoMonolog\Handler\LogzIoHandler:
+        arguments:
+           $token: '%env(LOGZ_IO_TOKEN)%'
```
Comment (OskarStark) [inline datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml:4]:
```suggestion
            $token: '%env(LOGZ_IO_TOKEN)%'
```

File: datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+services:
+    Inpsyde\LogzIoMonolog\Handler\LogzIoHandler:
+        arguments:
+            $token: '%env(LOGZ_IO_TOKEN)%'
+            $host: !php/const Inpsyde\LogzIoMonolog\Handler\LogzIoHandler::HOST_EU
```
Comment (OskarStark) [inline datana-gmbh/logz-io-handler/1.0/config/packages/datana_logz_io_handler.yaml:5]:
```suggestion
            $host: 'listener-eu.logz.io'
```

**Comment by OskarStark** [comment]
Thank you Tobias!

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**